examples/camera-app/linux/src/clusters/webrtc_provider/webrtc-provider-manager.cpp

```diff
@@ -219,21 +218,54 @@ WebRTCProviderManager::ValidateStreamUsage(StreamUsageEnum streamUsage,
 void WebRTCProviderManager::ScheduleAnswerSend()
 {
     DeviceLayer::SystemLayer().ScheduleLambda([this]() {
-        CHIP_ERROR err = SendAnswerCommand(mCurrentSessionId, mSdpAnswer, mPeerId, mOriginatingEndpointId);
+        ChipLogProgress(NotSpecified, "Sending Answer command to node " ChipLogFormatX64, ChipLogValueX64(mPeerId.GetNodeId()));
 
-        if (err != CHIP_NO_ERROR)
-        {
-            ChipLogError(NotSpecified, "Answer failed: %" CHIP_ERROR_FORMAT, err.Format());
-        }
+        mCommandType = CommandType::kAnswer;
 
-        mOriginatingEndpointId = 0;
-        mOriginatingEndpointId = 0;
-        mPeerId                = ScopedNodeId();
+        // Attempt to find or establish a CASE session to the target PeerId.
+        CASESessionManager * caseSessionMgr = Server::GetInstance().GetCASESessionManager();
+        VerifyOrDie(caseSessionMgr != nullptr);
+
+        caseSessionMgr->FindOrEstablishSession(mPeerId, &mOnConnectedCallback, &mOnConnectionFailureCallback);
```

Comment:
So if we need to send Answer to multiple different destinations we'll just kind of fail here (in the sense that we will never send it to the first one if we get this called for the second one before CASE setup for the first one is done)?